R/db-odbc-mssql.R

```diff
@@ -68,7 +68,28 @@
       as.double     = sql_cast("NUMERIC"),
       as.character  = sql_cast("VARCHAR(MAX)"),
       log           = sql_prefix("LOG"),
+      nchar         = sql_prefix("LEN"),
+      atan2         = sql_prefix("ATN2"),
+      substr        = sql_prefix("SUBSTRING"),
+      ceil          = sql_prefix("CEILING"),
       ceiling       = sql_prefix("CEILING"),
+      # Casting as BIT converts the Integer result into an actual logical
```

Comment:
What version of SQL server are you using? [This page](https://docs.microsoft.com/en-us/sql/t-sql/queries/is-null-transact-sql) suggests that `IS NULL` should work.

Does it only work in WHERE statements?